__tests__/app.test.js

```diff
@@ -43,3 +43,38 @@ describe('GET api', () => {
             })
     })
 });
+
+describe('GET api/articles/:article_id', () => {
+    test('sending an API request to api/articles/1 returns a Status-200: responds with an object, with a key articles, which has the value of the article matching the given id - in this case - 1.', () => {
+        return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.article).toMatchObject({
+                    "author": expect.any(String),
+                    "title": expect.any(String),
+                    "article_id": expect.any(Number),
+                    "body": expect.any(String),
+                    "created_at": expect.any(String),
+                    "votes": expect.any(Number),
+                    "article_img_url": expect.any(String)
+                })
```

Comment:
Brilliant! But we should be checking that the article_id is 1 here not any number :)